pkg/constants/constants.go

```diff
@@ -16,4 +16,19 @@ const (
 	WeChat   = "wechat"
 
 	Namespace = "namespace"
+
+	AlertFiring   = "firing"
+	AlertResolved = "resolved"
+
+	AlertName      = "alertname"
+	AlertType      = "alertype"
+	AlertTime      = "alertime"
+	AlertMessage   = "message"
+	AlertSummary   = "summary"
+	AlertSummaryCN = "summaryCn"
```

Comment:
summaryCN

Reply:
The event rule use `summaryCn`.

https://github.com/kubesphere/kube-events/blob/master/helm/templates/crs/cluster-rules-default.yaml#L33